src/shogun/evaluation/ROCEvaluation.cpp

```diff
@@ -19,15 +19,16 @@ CROCEvaluation::~CROCEvaluation()
 
 float64_t CROCEvaluation::evaluate(CLabels* predicted, CLabels* ground_truth)
 {
-	return evaluate_roc(predicted,ground_truth);
+	REQUIRE(predicted->get_label_type()==LT_BINARY, "ROC evalution requires binary labels.");
```

Comment:
Sorry to be picky here:

 * We need a line break (>80chars)
 * There is a newline missing at the end of the string.
 * The message should give information what went wrong. `"Given labels (%d) must be binary (%d).\n", predicted->get_label_type(), LT_BINARY)`